File: examples/notes/read/go/readANote.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/notes"
	"os"
)

// Read a note
func ReadAnote() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.Read(&notes.ReadRequest{
		Id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
	})
	fmt.Println(rsp, err)
}

File: examples/js/notes/read/readANote.js
const { NotesService } = require("m3o/notes");

// Read a note
async function readAnote() {
  let notesService = new NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.read({
    id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
  });
  console.log(rsp);
}

readAnote();
